manifest.md

```diff
@@ -1,16 +1,22 @@
-*ЗОЖ МАНИФЕСТ*
+*ЗОЖ ХУЁЖ МАНИФЕСТ*
 
-* Бегать 4 раза в неделю — это хорошо. Иногда и реже норм, если уж обстоятельства такие. Иногда даже и совсем не бегать норм, если болеешь, например. Думай головой.
-* В качалку — тоже хорошо. Ну смотри сам, как с бега время будет оставаться.
-* Пить в одного — плохо. Хотя изредка и это норм.
-* Пить вдвоём часто — плохо. Но допустимы исключения — иногда это норм.
-* Пить в большой компании это, в принципе, норм, особенно если почти всегда в нескольких определённых. А вот пить постоянно в разных больших компаниях — плохо. Это как искать повод напиться, ища каждый день разные митапы только для отмазки.
-* Курить — плохо. Даже иногда. Может совсем изредка это и норм, но решай сам.
-* Обжираться — плохо. Целая курица на обед — это слишком много. Половина — норм. Целая порция индийской еды из Tandoori Express  — это слишком много. Половина — норм. Иногда можно и целую, но редко.
-* Пить часто колу из бесплатного холодильника в офисе — плохо. Иногда — норм. 
-* Хватит путешествовать почти каждые выходные. Ты в Амстердаме на выходных только три раза был в этом году. Помнишь же, хотел в этом году меньше путешествовать. Слишком частые отъезды притупляют удовольствие от путешествий, сбивают графики спорта, способствуют большему потреблению алкоголя, да ещё и кучу денег сжигают. Путешествуй иногда, для удовольствия, развития и отдыха, а не от нечего делать,  и — уж тем более — не для сегментов.
-* Не следить за здоровьем — плохо. Помни об этом. А дальше уж знаешь что делать.
+* Бегать 4 раза в неделю — это заебись. Иногда и реже норм, если уж обстоятельства такие. Иногда даже и совсем не бегать норм, если болеешь, например. Думай головой.
+* В качалку — тоже заебись. Ну смотри сам, как с бега время будет оставаться.
+* Пить в одного — хуёво. Хотя изредка и это норм.
+* Пить вдвоём часто — хуёво. Но допустимы исключения — иногда это норм.
+* Воображаемые друзья это заебись. Но пить с ними - хуёво.
+* Пить в большой компании это, в принципе, норм, особенно если почти всегда в нескольких определённых. А вот пить постоянно в разных больших компаниях — хуёво. Это как искать повод напиться, ища каждый день разные митапы только для отмазки.
+* Курить — хуёво. Даже иногда. Может совсем изредка это и норм, но решай сам.
+* Курить траву - заебись. Ты же в Амстердаме, там лигалайз. А лигалайз это заебись.
```

Comment:
Your dashes are different from long dashes in the original, this makes it inconsistent.